doc/contributor/workflow.md

```diff
@@ -63,6 +63,18 @@ $ echo MX_BINARY_SUITES=truffle,sdk > mx.truffleruby/env
 $ jt build
 ```
 
+
+
+By default the `jt build` command only builds the JVM-based launcher for TruffleRuby.
+If you'd like to build the native image, you can do so by providing an extra argument
+to the build command.
+
+```bash
+$ jt build native
+```
+
+This will create a new file name `native-ruby` in your `bin/` directory.
+
```

Comment:
Sorry to be a pain, but could you mention SubstrateVM so it's google-able, mention it will download something, and get rid of those random two extra lines?